src/index.html

```diff
@@ -13,20 +16,33 @@
     <title>Document</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="/src/style.css"
     />
```

Comment:
The attributes of the `<link>` tag should be indented with 2 spaces. The closing bracket should be aligned with the opening tag.